android/src/main/java/nandorojo/modules/galeria/EdgeToEdgeImageViewerDialogFragment.kt

```diff
@@ -0,0 +1,26 @@
+package nandorojo.modules.galeria
+
+import android.app.Dialog
+import android.os.Bundle
+import androidx.core.view.WindowCompat
+import androidx.core.view.WindowInsetsControllerCompat
+import com.github.iielse.imageviewer.ImageViewerDialogFragment
+
+class EdgeToEdgeImageViewerDialogFragment(private val isAppearanceLightSystemBars: Boolean) :
+    ImageViewerDialogFragment() {
+
+    override fun onCreateDialog(savedInstanceState: Bundle?): Dialog {
+        return Dialog(requireActivity(), R.style.Theme_FullScreenDialog).apply {
+            setCanceledOnTouchOutside(true)
+
+            window?.let {
```

Comment:
Here we enters edge-to-edge, and set the system bars colors based on the chosen theme. It's safe as it operates in a different `Window` (compared to `MainActivity`), so it will not mess with `SystemBars`, `StatusBar`, etc.